# contracts/test/MockERC20Confidential.sol  ->  contracts/test/MockERC20Confidential.fsol
# upstream 5138cb8 .. ported 86f1b7c

diff --git a/contracts/test/MockERC20Confidential.sol b/contracts/test/MockERC20Confidential.fsol
index 10051d0..ad45e54 100644
--- a/contracts/test/MockERC20Confidential.sol
+++ b/contracts/test/MockERC20Confidential.fsol
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.25;
 
-import { ERC20Confidential } from "../ERC20Confidential/ERC20Confidential.sol";
+import { ERC20Confidential } from "../ERC20Confidential/ERC20Confidential.fsol";
 
 /**
  * @dev Mock implementation of ERC20Confidential for testing purposes with configurable decimals.
